src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -140,7 +148,7 @@ protected void move(final boolean nonCombat, final IMoveDelegate moveDel, final
     BattleCalculator.clearOOLCache();
     ProLogUI.notifyStartOfRound(data.getSequence().getRound(), player.getName());
     initializeData();
-    ProBattleUtils.setData(data);
+    calc.setData(data);
```

Comment:
That doesn't answer the question. What usage of the calc required the set on line 151?